my-app/src/pages/Homepage/Homepage.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from 'react'
+import "./Homepage.css"
+import Nav from '../../components/Nav/Nav'
+import hero_img from "../../assets/home_img.jpg"
+import { Link } from 'react-router-dom'
+import ViCard from '../../components/VideoCard/ViCard'
+import axios from "axios"
+
+
+const Homepage = () => {
+  const [showVideoList, setShowVideoList] = useState([]);
+  useEffect(() => {  
+  (async () =>{
+   try {
+     const resp = await axios.get(`/api/videos`)
+     setShowVideoList(resp.data.videos)
+     console.log(resp.data.videos)
+   } catch (error) {
+     console.log(error)
+   }
+  })();
+  },[])
+  
+  return (
+    <div className='container_home'>
+      <Nav/>
+      <div className='hero_img_container'>
+        <img className='hero_img_width' src={hero_img} alt=""  />
```

Comment:
alt text could be added, just nitpicking for now not very p0.

Reply:
oh..I missed.. will do that